### ISSUE TITLE
--exclude doesn't work with relative directory

### ISSUE THREAD

**Issue by superarts** [nicklockwood/SwiftFormat#903]
I may be missing the obvious here, but commands like this wouldn't work for me:

```
swiftformat ../../../MyProject/ --exclude ../../../MyProject/Pods/ --lint
```

While this works:

```
swiftformat ../../../MyProject/ --exclude ~/prj/ios/MyProject/Pods/ --lint
```

I'm pretty sure I didn't make any human error here, as `ls ../../../MyProject` gives me the expected result. I can't see it from the open issues; could anyone confirm if it's a real issue please? I'm on 0.47.12 `68cac6de69aafb44b60ae49a919600966b0dcabe`.

**Comment by nicklockwood** [comment]
@superarts this looks like a bug - paths containing ../ aren't correct matched. I'll fix it asap.

**Comment by superarts** [comment]
Thanks!

**Comment by nicklockwood** [comment]
@superarts fixed in 0.48.0
